config/initializers/extra_extra.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+ExtraExtra.renderer_options = {
+  header_level_offset: 1, # h1's are h1
```

Comment:
My understanding is that the only `h1` should be your home link, and page headings should start at h2.

Reply:
Fixed.